### ISSUE TITLE
Not processing short train segment

### ISSUE THREAD

**Issue by thohug** [datamapio/posmo_one_testing#15]
**Setting**
Zurich, Switzerland, Android

wifi-fence: 1
do_not_track: 0
wifi_over_gps: 1
data_over_wifi_only: 1

**Issue**
Processed segements are missing out on train section (yellow).

![grafik](https://user-images.githubusercontent.com/67195528/89202885-eb31e780-d5b3-11ea-8dce-b4b15c761f5a.png)

**Comment by rogerfischer** [comment]
Have you downloaded the newest version of the app?

**Comment by thohug** [comment]
> 
> 
> Have you downloaded the newest version of the app?

No, thanks for the reminder. btw got really good results today on the train to Bern (where it otherwise was rather sketchy...) - everything almost perfecty recognised including bus and bike ride.

**Comment by thohug** [comment]
same today; the 2 measurements in the train (Oerlikon - Wiedikon via main station) weren't enough to process the train segment.

![grafik](https://user-images.githubusercontent.com/67195528/89264797-d432da00-d633-11ea-83b0-c08ae6c4362a.png)
